frontend/src/utils/constants.js

```diff
@@ -0,0 +1,2 @@
+export const API_BASE_URL = `http://127.0.0.1:8000/api/v1/`;
+export const userId = 1;
```

Comment:
Hopefully you are not planning on hardcoding `userId`?

Reply:
UserData() func is not implemented yet and will update on next merge:

`export const userId = UserData()?.user_id;`